content/3_Silver/Sorting_Custom.mdx

```diff
@@ -591,4 +592,10 @@ public class Sol {
 
 ## Problems
 
+<Info>
+
+Sorting is a very general idea, so many of the problems below may use other Silver concepts, such as [prefix sums](/silver/prefix-sums), or [binary search](/silver/binary-search-ans), etc.
```

Comment:
binsearch-ans isn't a prerequisite though :o

Reply:
I don't think it really makes sense to have a prereq that isn't very involved with sorting. Wouldn't it be sufficient just to say that the problems below may use other silver concepts?

Alternative, we can move this module to the end so it's more in sequential order.

Reply:
> I don't think it really makes sense to have a prereq that isn't very involved with sorting. Wouldn't it be sufficient just to say that the problems below may use other silver concepts?

Sure (and just include prefix sums)?